Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,22 +1,15 @@
 import React from 'react';
-import './App.css';
+import CalculatorLayout from './components/CalculatorLayout/CalculatorLayout';
+
+
+
+const numArray = [1, 2, 3, 4, 5, 6, 7, 8, 9];
```

Comment:
although trivial. this number order won't generate a typical calculator button layout when mapped.